lambda/update/parse/changes.go

```diff
@@ -123,6 +142,43 @@ func (p *Parser) Field(key string, v any, opts ...Option) *Parser {
 	return p
 }
 
+func oldEqualsExisting(old any, existing any) bool {
+	if v, ok := existing.(*time.Time); ok {
+		if old == nil {
+			return v.IsZero()
+		}
+
+		RFC3339local := "2006-01-02T15:04:05Z"
+		return old.(string) == v.Format(RFC3339local)
```

Comment:
I was having a bit of nightmare with time formats - there is probably a more robust way to check equality here.

Reply:
you could parse old, then use `t1.Equal(t2)`

Reply:
I'd tried this initially but couldn't work out the right format to parse `old` in as `v` is typecast with the format `2024-04-17 10:42:32.628328 +0100 m=-86399.999282124`